Trivia_Trainer.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <meta charset="UTF-8">
+    <head>
+        <title>Trivia_Trainer Gameplay Page</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="<link rel=stylesheet href="https://s3-us-west-2.amazonaws.com/colors-css/2.2.0/colors.min.css">"
```

Comment:
This link is formatted incorrectly as well as having a trailing quotation mark that shows up on the page.